client/my-sites/design-menu/design-tools/control-list.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+
+const ControlButton = props => (
+	<div onClick={ props.onClick } className="design-menu-controls__control-button">
+		<span className="design-menu-controls__control-button__title">{ props.title }</span>
+		<Gridicon icon="chevron-right" size={ 24 } className="design-menu-controls__control-button__arrow" />
+	</div>
+);
+
+const ControlList = React.createClass( {
```

Comment:
It'd be good to have a UI concept before diving too deep into the implementation of controls. I thought we could even avoid controls completely in favor of general custom components.